sdk/monitor/Azure.Monitor.Query/src/MigrationGuide.md

```diff
@@ -0,0 +1,173 @@
+
+### Cross-service SDK improvements
+
+The `Azure.Monitor.Query` client library also takes advantage of the cross-service improvements made to the Azure development experience. Examples include:
+
+- Using the new [`Azure.Identity`](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity#readme) library to share a single authentication approach between clients.
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries.
+
+### New features
+
+There are a variety of new features in version 1.0 of the `Azure.Monitor.Query` library. Some highlights include:
+
+- The ability to execute a batch of Kusto queries with the `LogsQueryClient.QueryBatch()` API.
+- The ability to configure the retry policy used by the operations on the client.
+- The ability to map a Logs query result to a strongly typed model.
+- The ability to retrieve by column name instead of by column index.
+- Authentication with Azure Active Directory (Azure AD) credentials using [`Azure.Identity`](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/identity/Azure.Identity#readme).
+
+For more new features, changes, and bug fixes, see the [CHANGELOG](https://github.com/Azure/azure-sdk-for-net/blob/Azure.Monitor.Query_1.0.1/sdk/monitor/Azure.Monitor.Query/CHANGELOG.md).
+
+## Important changes
+
+### The client
+
+To provide a more intuitive experience, the top-level client to query logs was renamed to `LogsQueryClient` from `OperationalInsightsDataClient`. `LogsQueryClient` can be authenticated using Azure AD. This client is the single entry point to execute a single Kusto query or a batch of Kusto queries.
+
+#### Consistency
+
+There are now methods with similar names, signatures, and locations to create senders and receivers. The result is consistency and predictability on the various features of the library.
+
+### Client constructors and authentication
+
+In `Microsoft.Azure.OperationalInsights` v1.1.0:
+
+```csharp
+using Microsoft.Azure.OperationalInsights;
+using Microsoft.Rest;
+using Microsoft.Rest.Azure.Authentication;
+	
+// code omitted for brevity
+
+var adSettings = new ActiveDirectoryServiceSettings
+	{
+	    AuthenticationEndpoint = new Uri("https://login.microsoftonline.com"),
+	    TokenAudience = new Uri("https://api.loganalytics.io/"),
+	    ValidateAuthority = true
+	};
+	
+ServiceClientCredentials credentials = ApplicationTokenProvider.LoginSilentAsync(
+	    "<domainId or tenantId>", "<clientId>", "<clientSecret>", adSettings).GetAwaiter().GetResult();
+var client = new OperationalInsightsDataClient(credentials);
+```
+
+In `Azure.Monitor.Query` v1.0.x:
+
+```csharp
+using Azure.Monitor.Query;
+using Azure.Identity;
+
+// code omitted for brevity
+
+var client = new LogsQueryClient(new DefaultAzureCredential());
```

Comment:
The `Microsoft.Azure.OperationalInsights` code sample uses a tenant ID, client ID, and client secret to authenticate. Let's use `ClientSecretCredential` instead of `DefaultAzureCredential` to better show how that part translates. I think you'd have something like this:

```suggestion
var credential = new ClientSecretCredential("<domainId or tenantId>", "<clientId>", "<clientSecret>");
var client = new LogsQueryClient(credential);
```